cpp/src/partitioning/partitioning.cu

```diff
@@ -16,9 +16,10 @@
 
 #include <cudf/column/column_factories.hpp>
 #include <cudf/copying.hpp>
+#include <cudf/detail/gather.cuh>
```

Comment:
Just to note I need this header now because this file uses `gather_bitmask` which isn't exposed via the `gather.hpp` header, only `gather.cuh`.